website/content/maintainers/github/templates/required/governance-subprojects.md

```diff
@@ -0,0 +1,290 @@
+---
+title: "The Governance By Subprojects Template"
+linkTitle: "Subproject Governance"
+date: 2022-07-14
+draft: true
+weight: 33
+---
+
+# Governance By Subprojects
```

Comment:
This looks fine but is missing the common structure that I thought we would use for each template howto:

See https://contribute.cncf.io/maintainers/github/templates/required/contributing/ for an example:

header that explains who must use the template
![Screen Shot 2022-09-01 at 4 31 36 PM](https://user-images.githubusercontent.com/1368985/188016724-a21a7531-6be4-4399-9bfe-e0773a044b0a.png)

explanation of conventions used in the template and how they should fill it out (e.g. do they need to look for TODO, html comments, icons, etc)
![Screen Shot 2022-09-01 at 4 32 19 PM](https://user-images.githubusercontent.com/1368985/188016913-75adbd64-fb2c-4720-bf1d-6053deb66821.png)

Reply:
OK, we can add that.

Reply:
Er ... where does that graphic come from?

Reply:
That graphic is a screenshot of the contributor's guide template, https://github.com/cncf/project-template/blob/main/CONTRIBUTING.md#ask-for-help. Each section of the template links to the howto in the instructions lik below the section title and then any tasks are called out with that warning emoji and bold todo text.

https://github.com/cncf/tag-contributor-strategy/blob/main/website/content/maintainers/github/templates/sample-instructions.png

Reply:
It looks very different when I look at it in my browser, so I'm just going to re-use the screenshot